cmd/jobsList.go

```diff
@@ -62,18 +74,21 @@ func init() {
 	}
 
 	jobsCmd.AddCommand(lsCmd)
+
+	jobsCmd.PersistentFlags().StringVar(&commandLineInput.withStatus, "with-status", "All",
+		"only remove the jobs with this status, available values: Cancelled, Completed, Failed, InProgress, All")
```

Comment:
>remove [](start = 8, length = 6)

Remove? Shouldn't this be "list"?

Reply:
Yes, it should've. I'll put this in the next commit.